src/main/resources/locales/en/view-option.json

```diff
@@ -1,8 +1,10 @@
 {
   "title": "View Option",
   "radioLegend": "What would you like to view?",
-  "radio1": "Court or Tribunal publications",
+  "radio1": "Court or tribunal hearings",
```

Comment:
Is the AC in the ticket wrong?
https://tools.hmcts.net/jira/browse/PUB-888
"Court or Tribunal hearings" to be "Court or tribunal hearings"